lib/router.js

```diff
@@ -23,20 +23,30 @@ module.exports = class Router {
             return;
         }
 
-		const cmd = matches[1].toLowerCase();
+		const cmd = matches[1].toLowerCase();        
         data['cmd'] = cmd;
-        console.log(cmd, data['cmd']);
         data['user_text'] = data.text.replace(matches[0], '').trim();
         data['args'] = data.text.match(argPattern);
 
+        
+        const module = this.resolveComamnd(cmd);        
+        if (data.args != null 
+            && (data.args.includes('-help') || data.args.includes('--help'))) {
+            this.bot.postMessage(data.channel, module.help());                            
+            return;
+        }
+
+        module.handle(data);        
+	}
+
+
+    resolveComamnd(cmd) {    
```

Comment:
Unrelated but I need to figure out how to make aliases or custom `cmds` for classes